pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java

```diff
@@ -423,22 +423,22 @@ public void properties() throws Throwable {
                 .allowedClusters(allowedClusters)
                 .build();
         response = asynRequests(ctx -> properties.createTenant(ctx, "test-property", tenantInfo));
-        verify(properties, times(2)).validateSuperUserAccess();
+        verify(properties, times(1)).validateSuperUserAccess();
```

Comment:
Why do we change here?

Reply:
Because we use ``validateSuperUserAccessAsync`` to instead of ``validateSuperUserAccess``. 
So, the times counter is changed.

Reply:
Do we need to add the verify for `validateSuperUserAccessAsync`?

Reply:
why not?

Reply:
I mean I think we need to add that verification. I think we'd better add another verification for `validateSuperUserAccessAsync ` after line 450: `verify(properties, times(2)).validateSuperUserAccessAsync();`. This will make the test more clear.

Reply:
Agree with you +1
I will change this part. Thanks for your suggestions. : )